acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/AbstractConsumablePage.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.digital.ps.test.acceptance.pages;
+
+import uk.nhs.digital.ps.test.acceptance.webdriver.WebDriverProvider;
+
+abstract class AbstractConsumablePage extends AbstractPage {
+
+    static final String URL = "http://localhost:8080";
```

Comment:
Can't we use DI to avoid the need to hard-code this, and thus the need for this AbstractConsumablePage to exist at all?

Reply:
We've discussed and will leave as is for now